components/formats-gpl/src/loci/formats/in/SISReader.java

```diff
@@ -156,8 +156,8 @@ protected void initStandardMetadata() throws FormatException, IOException {
     long metadataPointer = ifd.getIFDLongValue(SIS_TAG, 0);
     in.seek(metadataPointer);
 
-	// The tag here is interpreted according to the implementation in tiff.py (https://gist.github.com/tbenst/7db5a9aa9b221e67ba85c5642b23f094)
-	// '<4s6shhhhh6s32sh', 60 bytes in total  => (magic, _, minute, hour, day, month, year, _, name, tagcount)
+	  // The tag here is interpreted according to the implementation in tiff.py (https://gist.github.com/tbenst/7db5a9aa9b221e67ba85c5642b23f094)
+	  // '<4s6shhhhh6s32sh', 60 bytes in total  => (magic, _, minute, hour, day, month, year, _, name, tagcount)
```

Comment:
Here and in a couple of lines below the indentation does not seem to be right (overindented)